File: core-examples/src/main/js/io/vertx/example/core/eventbus/pointtopoint/sender.js
var eb = vertx.eventBus();

// Send a message every second

vertx.setPeriodic(1000, function (v) {

  eb.send("ping-address", "ping!", function (reply, reply_err) {
    if (reply_err == null) {
      console.log("Received reply " + reply.body());
    } else {
      console.log("No reply");
    };
  });

});

File: core-examples/src/main/rb/io/vertx/example/core/eventbus/pointtopoint/sender.rb
eb = $vertx.event_bus()

# Send a message every second

$vertx.set_periodic(1000) { |v|

  eb.send("ping-address", "ping!") { |reply_err,reply|
    if (reply_err == nil)
      puts "Received reply #{reply.body()}"
    else
      puts "No reply"
    end
  }

}
